src/index.js

```diff
@@ -0,0 +1,104 @@
+import express from 'express';
+
+const app = express();
+const PORT = process.env.PORT || 5000;
+
+
+ app.use(express.json());
+
+
+const database = {
+
+		entries: [{
+				title: "heading", message: "message"
+		}],
+
+		users: [{
+				firstname: "first", lastname: "last", email: "email", password: "password"
+		}],
+
+};
+
+
+app.post('/api/v1/entries', (req, res) =>{
+	
+	const { title, message } = req.body;
+	if (title === ' ' || message === ' ' ) {
+		res.status(422).json('Please fill fields');
+	} 
+	 else if (title && message) {
+		database.entries.push(req.body);
+		res.status(201).json(database.entries);
+	} 
+	else {
+		res.status(400).json('Bad Request');
+	}
+	
+});
+
+
+
+app.post('/api/v1/signup', (req, res) =>{
+	console.log(req.body);
+	const { firstname, lastname, email, password } = req.body;
+	if (firstname === ' ' || lastname === ' ' || email === ' ' || password === ' ') {
+		res.json('Please fill fields').status(422);
+	} else if (firstname && lastname && email && password) {
+		const user = database.users.filter(u => u.email === email && u.password === password);
+		if (user.length > 0 && user[0].email) {
+			res.status(409).json('Email and password already taken!');
+		} else {
+			database.users.push(req.body);
+			res.status(201).json(database.users);
+		}	
```

Comment:
It is not ideal to log to the console. Again, your variable names should be in camel case.

Reply:
I have removed the console